usaspending_api/financial_activities/migrations/0001_initial.py

```diff
@@ -78,7 +78,7 @@ class Migration(migrations.Migration):
             name='TasProgramActivityObjectClassQuarterly',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'DATA Act Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
+                ('data_source', models.TextField(choices=[('USA', 'USAspending'), ('DBR', 'Data Broker')], help_text='The source of this entry, either Data Broker (DBR) or USASpending (USA)', null=True)),
```

Comment:
This TextField needs to be updated in a separate migration.

Reply:
same as above